examples/js/e-commerce-umd/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "example-instantsearch-ecommerce-umd",
+  "version": "1.22.0",
+  "private": true,
+  "scripts": {
+    "start": "BABEL_ENV=parcel parcel index.html --port 3000",
+    "build": "BABEL_ENV=parcel parcel build index.html --public-url .",
+    "website:examples": "BABEL_ENV=parcel parcel build index.html --public-url . --dist-dir=../../../website/examples/js-umd/e-commerce"
```

Comment:
technically this means it's also accessible from our main docs, but that's ok, we can document how it works that way